spec/pundit/frodo_policy_spec.rb

```diff
@@ -152,6 +171,21 @@ def salido_pos?
       end
     end
 
+    context '#salido_bridge?' do
```

Comment:
@aperezmontan you know more about how we store these records than I do-- can the bridge ever be a client application instead of a user?

Reply:
The bridge will always be a client application